tests/testsuite/help.rs

```diff
@@ -147,7 +151,7 @@ fn help_alias() {
     // The `empty-alias` returns an error.
     cargo_process("help empty-alias")
         .env("PATH", Path::new(""))
-        .with_stderr_contains("[..]The subcommand 'empty-alias' wasn't recognized[..]")
+        .with_stderr_data("[..] The subcommand 'empty-alias' wasn't recognized [..]")
```

Comment:
Should we drop the `[..]`s?  We are trying to move away from them where it makes sense because they get in the way of snapshot updating

Reply:
Removed in e5e9f2f, but in fact there is another issue #14076 with this test case. I mentioned that as well in the same commit.